easybuild/tools/toolchain/toolchain.py

```diff
@@ -317,6 +317,10 @@ def is_dep_in_toolchain_module(self, name):
         """Check whether a specific software name is listed as a dependency in the module for this toolchain."""
         return any(map(lambda m: self.mns.is_short_modname_for(m, name), self.toolchain_dep_mods))
 
+    def _pre_prepare(self):
+        """Toolchain-specific preparations thay should be done first."""
```

Comment:
either rename the method to indicate what is claimed  in the docstring (toolchain specific preparation in the prepare method of the toolchain module, who would have tought that...), but better just override the `prepare` method in the `CrayPEWrapper`.